### PR TITLE
Add Prometheus integration and metric descriptions

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

File: content/rs/administering/monitoring-metrics/prometheus-integration.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
+---
+Title: Redis Enterprise Software Integration with Prometheus
+description: 
+weight: $weight
+alwaysopen: false
+---
+Redis Enterprise lets you connect your Prometheus or Grafana server to your Redis Enterprise cluster in order to collect and display metrics data. Metrics are exposed at the node, database, shard and proxy levels.
```
Comment (danni-m) [inline content/rs/administering/monitoring-metrics/prometheus-integration.md]: @bmansheim @Paz-Yanover Grafana doesn't connect to Redis Enterprise, its querying Prometheus.

File: content/rs/administering/monitoring-metrics/prometheus-integration.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+        - targets: ['localhost:9090']
+
+    # scrape Redis Enterprise
+    - job_name: 'redis-enterprise'
+        scrape_interval: 30s
+        scrape_timeout: 30s
+        metrics_path: /
+        scheme: https
+        tls_config:
+        insecure_skip_verify: true
```
Comment (danni-m) [inline content/rs/administering/monitoring-metrics/prometheus-integration.md]: @bmansheim missing tab? please check that this configuration file works before pushing.

File: content/rs/administering/monitoring-metrics/prometheus-integration.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+    1. Login to Grafana. If you installed Grafana locally, go to `localhost:3000` and login with:
+
+    	Username: admin
+    	Password: secret
+
+1. Click **Data Sources**.
+
+    ![data-sources](/images/rs/data-sources.png?width=300)
+
+1. Add a new data source with:
```
Comment (danni-m) [inline content/rs/administering/monitoring-metrics/prometheus-integration.md]: @bmansheim you can link to grafana docs to have more details.

File: content/rs/administering/monitoring-metrics/prometheus-integration.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+## Quick Start 
+
+To get started with custom monitoring:
+
+1. If you don't already have Prometheus and Grafana installed, setup the Prometheus and Grafana servers:
+    1. You can use this _docker-compose.yml_ file to define the containers:
+
+        ```
+        version: '3'
```
Comment (bmansheim) [inline content/rs/administering/monitoring-metrics/prometheus-integration.md]: @danni-m This docker compose file doesn't work for me. Can you check it?

**Pull request merged**